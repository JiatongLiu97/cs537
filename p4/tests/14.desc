[128] Allocate multiple pages that will make 32 extra pages evicted with CLOCKSIZE = 128
