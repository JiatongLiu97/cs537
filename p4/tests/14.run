cd src;
rm -rf param_old.h;
mv param.h param_old.h;
sed -E 's/((^|(\t| ))*CLOCKSIZE)(\t| )*[^ ]*/\3CLOCKSIZE\t128/' param_old.h > param.h;
make -f Makefile.test clean > /dev/null 2>&1;
make -f Makefile.test xv6.img > /dev/null 2>&1;
make -f Makefile.test fs.img > /dev/null 2>&1;
../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_14 | grep XV6_TEST_OUTPUT; 
cd ..
