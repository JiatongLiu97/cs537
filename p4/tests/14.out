XV6_TEST_OUTPUT Index 0: pdx: 0x0, ptx: 0xA0, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 1: pdx: 0x0, ptx: 0x9F, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 2: pdx: 0x0, ptx: 0x9E, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 3: pdx: 0x0, ptx: 0x9D, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 4: pdx: 0x0, ptx: 0x9C, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 5: pdx: 0x0, ptx: 0x9B, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 6: pdx: 0x0, ptx: 0x9A, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 7: pdx: 0x0, ptx: 0x99, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 8: pdx: 0x0, ptx: 0x98, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 9: pdx: 0x0, ptx: 0x97, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 10: pdx: 0x0, ptx: 0x96, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 11: pdx: 0x0, ptx: 0x95, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 12: pdx: 0x0, ptx: 0x94, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 13: pdx: 0x0, ptx: 0x93, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 14: pdx: 0x0, ptx: 0x92, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 15: pdx: 0x0, ptx: 0x91, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 16: pdx: 0x0, ptx: 0x90, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 17: pdx: 0x0, ptx: 0x8F, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 18: pdx: 0x0, ptx: 0x8E, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 19: pdx: 0x0, ptx: 0x8D, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 20: pdx: 0x0, ptx: 0x8C, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 21: pdx: 0x0, ptx: 0x8B, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 22: pdx: 0x0, ptx: 0x8A, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 23: pdx: 0x0, ptx: 0x89, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 24: pdx: 0x0, ptx: 0x88, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 25: pdx: 0x0, ptx: 0x87, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 26: pdx: 0x0, ptx: 0x86, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 27: pdx: 0x0, ptx: 0x85, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 28: pdx: 0x0, ptx: 0x84, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 29: pdx: 0x0, ptx: 0x83, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 30: pdx: 0x0, ptx: 0x82, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 31: pdx: 0x0, ptx: 0x81, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 32: pdx: 0x0, ptx: 0x80, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 33: pdx: 0x0, ptx: 0x7F, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 34: pdx: 0x0, ptx: 0x7E, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 35: pdx: 0x0, ptx: 0x7D, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 36: pdx: 0x0, ptx: 0x7C, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 37: pdx: 0x0, ptx: 0x7B, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 38: pdx: 0x0, ptx: 0x7A, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 39: pdx: 0x0, ptx: 0x79, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 40: pdx: 0x0, ptx: 0x78, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 41: pdx: 0x0, ptx: 0x77, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 42: pdx: 0x0, ptx: 0x76, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 43: pdx: 0x0, ptx: 0x75, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 44: pdx: 0x0, ptx: 0x74, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 45: pdx: 0x0, ptx: 0x73, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 46: pdx: 0x0, ptx: 0x72, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 47: pdx: 0x0, ptx: 0x71, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 48: pdx: 0x0, ptx: 0x70, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 49: pdx: 0x0, ptx: 0x6F, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 50: pdx: 0x0, ptx: 0x6E, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 51: pdx: 0x0, ptx: 0x6D, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 52: pdx: 0x0, ptx: 0x6C, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 53: pdx: 0x0, ptx: 0x6B, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 54: pdx: 0x0, ptx: 0x6A, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 55: pdx: 0x0, ptx: 0x69, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 56: pdx: 0x0, ptx: 0x68, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 57: pdx: 0x0, ptx: 0x67, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 58: pdx: 0x0, ptx: 0x66, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 59: pdx: 0x0, ptx: 0x65, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 60: pdx: 0x0, ptx: 0x64, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 61: pdx: 0x0, ptx: 0x63, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 62: pdx: 0x0, ptx: 0x62, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 63: pdx: 0x0, ptx: 0x61, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 64: pdx: 0x0, ptx: 0x60, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 65: pdx: 0x0, ptx: 0x5F, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 66: pdx: 0x0, ptx: 0x5E, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 67: pdx: 0x0, ptx: 0x5D, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 68: pdx: 0x0, ptx: 0x5C, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 69: pdx: 0x0, ptx: 0x5B, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 70: pdx: 0x0, ptx: 0x5A, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 71: pdx: 0x0, ptx: 0x59, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 72: pdx: 0x0, ptx: 0x58, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 73: pdx: 0x0, ptx: 0x57, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 74: pdx: 0x0, ptx: 0x56, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 75: pdx: 0x0, ptx: 0x55, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 76: pdx: 0x0, ptx: 0x54, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 77: pdx: 0x0, ptx: 0x53, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 78: pdx: 0x0, ptx: 0x52, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 79: pdx: 0x0, ptx: 0x51, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 80: pdx: 0x0, ptx: 0x50, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 81: pdx: 0x0, ptx: 0x4F, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 82: pdx: 0x0, ptx: 0x4E, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 83: pdx: 0x0, ptx: 0x4D, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 84: pdx: 0x0, ptx: 0x4C, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 85: pdx: 0x0, ptx: 0x4B, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 86: pdx: 0x0, ptx: 0x4A, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 87: pdx: 0x0, ptx: 0x49, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 88: pdx: 0x0, ptx: 0x48, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 89: pdx: 0x0, ptx: 0x47, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 90: pdx: 0x0, ptx: 0x46, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 91: pdx: 0x0, ptx: 0x45, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 92: pdx: 0x0, ptx: 0x44, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 93: pdx: 0x0, ptx: 0x43, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 94: pdx: 0x0, ptx: 0x42, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 95: pdx: 0x0, ptx: 0x41, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 96: pdx: 0x0, ptx: 0x40, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 97: pdx: 0x0, ptx: 0x3F, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 98: pdx: 0x0, ptx: 0x3E, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 99: pdx: 0x0, ptx: 0x3D, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 100: pdx: 0x0, ptx: 0x3C, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 101: pdx: 0x0, ptx: 0x3B, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 102: pdx: 0x0, ptx: 0x3A, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 103: pdx: 0x0, ptx: 0x39, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 104: pdx: 0x0, ptx: 0x38, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 105: pdx: 0x0, ptx: 0x37, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 106: pdx: 0x0, ptx: 0x36, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 107: pdx: 0x0, ptx: 0x35, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 108: pdx: 0x0, ptx: 0x34, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 109: pdx: 0x0, ptx: 0x33, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 110: pdx: 0x0, ptx: 0x32, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 111: pdx: 0x0, ptx: 0x31, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 112: pdx: 0x0, ptx: 0x30, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 113: pdx: 0x0, ptx: 0x2F, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 114: pdx: 0x0, ptx: 0x2E, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 115: pdx: 0x0, ptx: 0x2D, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 116: pdx: 0x0, ptx: 0x2C, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 117: pdx: 0x0, ptx: 0x2B, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 118: pdx: 0x0, ptx: 0x2A, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 119: pdx: 0x0, ptx: 0x29, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 120: pdx: 0x0, ptx: 0x28, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 121: pdx: 0x0, ptx: 0x27, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 122: pdx: 0x0, ptx: 0x26, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 123: pdx: 0x0, ptx: 0x25, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 124: pdx: 0x0, ptx: 0x24, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 125: pdx: 0x0, ptx: 0x23, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 126: pdx: 0x0, ptx: 0x22, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 127: pdx: 0x0, ptx: 0x21, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 128: pdx: 0x0, ptx: 0x20, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 129: pdx: 0x0, ptx: 0x1F, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 130: pdx: 0x0, ptx: 0x1E, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 131: pdx: 0x0, ptx: 0x1D, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 132: pdx: 0x0, ptx: 0x1C, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 133: pdx: 0x0, ptx: 0x1B, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 134: pdx: 0x0, ptx: 0x1A, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 135: pdx: 0x0, ptx: 0x19, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 136: pdx: 0x0, ptx: 0x18, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 137: pdx: 0x0, ptx: 0x17, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 138: pdx: 0x0, ptx: 0x16, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 139: pdx: 0x0, ptx: 0x15, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 140: pdx: 0x0, ptx: 0x14, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 141: pdx: 0x0, ptx: 0x13, writable bit: 1, encrypted: 0, ref: 1
XV6_TEST_OUTPUT Index 142: pdx: 0x0, ptx: 0x12, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 143: pdx: 0x0, ptx: 0x11, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 144: pdx: 0x0, ptx: 0x10, writable bit: 1, encrypted: 1, ref: 0
XV6_TEST_OUTPUT Index 145: pdx: 0x0, ptx: 0xF, writable bit: 1, encrypted: 0, ref: 0
XV6_TEST_OUTPUT Index 146: pdx: 0x0, ptx: 0xE, writable bit: 1, encrypted: 0, ref: 1
